python/tests/plugin/test_worker_sdk.py

```
@@ -2209,6 +2310,24 @@ async def test_runtime_host_call_error_paths(host_stub: RecordingHostStub):
     with pytest.raises(WorkerSdkError, match="host call failed"):
         await runtime.emit_mark("mark")
 
+    class UnimplementedRuntimeDiagnosticsError(Exception):
+        def code(self) -> grpc.StatusCode:
+            return grpc.StatusCode.UNIMPLEMENTED
+
+    class OldHostStub:
+        async def GetRuntimeDiagnostics(self, request: Any) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `request`

(ANN401)